tsconfig.node.json

```diff
@@ -0,0 +1,10 @@
+{
+    "compilerOptions": {
+        "composite": true,
+        "skipLibCheck": true,
+        "module": "ESNext",
+        "moduleResolution": "bundler",
```

Comment:
Argument for '--moduleResolution' option must be: 'node', 'classic', 'node16', 'nodenext'.ts

Reply:
Ignoring this as not required